src/devices/cpu/hcd62121/hcd62121.h

```diff
@@ -108,6 +115,7 @@ class hcd62121_cpu_device :  public cpu_device
 	address_space *m_program;
 
 	int m_icount;
+	int m_cycles_until_timeout;
```

Comment:
This needs to be an explicitly sized type because it will need to be included in save states.  Also, don’t keep it with `m_icount` which is managed by the scheduler – put it with the other data members related to managing this.